pkg/exchange/okex/convert.go

```diff
@@ -256,18 +209,20 @@ func toLocalOrderType(orderType types.OrderType) (okexapi.OrderType, error) {
 }
 
 func toGlobalOrderType(orderType okexapi.OrderType) (types.OrderType, error) {
+	// Okex IOC and FOK only implement limit order
```

Comment:
"IOC, FOK are only allowed with limit order type, so we assume the order type is always limit order for FOK, IOC orders"